src/views/AView.ts

```diff
@@ -259,13 +259,39 @@ export abstract class AView extends EventHandler implements IView {
   }
 
   /**
-   * resolve the name of the current input selection
+   * resolve the id of the current input selection
    * @returns {Promise<string[]>}
    */
   protected resolveSelection(idType = this.idType): Promise<string[]> {
     return resolveIds(this.selection.idtype, this.selection.range, idType);
   }
 
+  /**
+   * resolve the name of the current input selection
+   * @returns {Promise<string[]>}
+   */
+  protected resolveSelectionByName(idType = this.idType): Promise<string[]> {
+    return resolveNames(this.selection.idtype, this.selection.range, idType);
+  }
+
+  /**
+   * resolves the ids of the current input selection to all 1:n related names, not just the first one like resolveSelection does
```

Comment:
```suggestion
   * resolve the ids of the current input selection to all 1:n related names, not just the first one like `resolveSelection` does
```

Either use always `resolve` or always `resolves`.